Tests/NIOTests/ByteBufferTest.swift

```diff
@@ -723,21 +723,16 @@ class ByteBufferTest: XCTestCase {
         XCTAssertEqual(otherBuf, buf)
 
         // Write to both buffers.
-        let firstResult = buf!.withUnsafeMutableWritableBytes { (ptr: UnsafeMutableRawBufferPointer) -> Bool in
-            XCTAssertEqual(cap, ptr.count)
-            memset(ptr.baseAddress!, 0, ptr.count)
-            return false
-        }
-        let secondResult = otherBuf!.withUnsafeMutableWritableBytes { (ptr: UnsafeMutableRawBufferPointer) -> Bool in
+        self.buf.setRepeatingByte(0, count: self.buf.capacity, at: 0)
```

Comment:
- `at: 0` isn't right, that should be `buf.writerIndex`
- `count: self.buf.capacity` also isn't right, that should be `buf.capacity - buf.writerIndex`